router/Navigator.js

```diff
@@ -28,13 +31,24 @@ const TabScreen = () => {
 };
 
 const StackScreen = () => {
+  const { isLoggedIn } = useContext(MainContext);
+
   return (
     <Stack.Navigator>
-      <Stack.Screen
-        name="Main"
-        component={TabScreen}
-        options={{ headerShown: false }}
-      ></Stack.Screen>
+      {isLoggedIn ? (
+        <>
+          <Stack.Screen
+            name="Main"
+            component={TabScreen}
+            options={{ headerShown: false }}
+          />
+        </>
+      ) : (
+        <>
+          <Stack.Screen name="Login" component={Login} />
```

Comment:
You can add the `{ headerShown: false}` so that it will remove the white container on the top